packages/vitest/src/constants.ts

```diff
@@ -1,11 +1,37 @@
 import { fileURLToPath } from 'url'
 import { resolve } from 'pathe'
+import type { UserConfig } from './types'
+import { defaults as coverageDefaults } from './integrations/coverage'
```

Comment:
We should move the default from coverage to this file, the module is running in different context.

Reply:
I've moved it there, but I still import it from here.